ui-participant/src/hub/HubPage.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+import HubPage from './HubPage'
+import { setupRouterTest } from '../test-utils/router-testing-utils'
+
+
+jest.mock('../providers/PortalProvider', () => {
+  return {
+    usePortalEnv: () => ({
+      portal: {
+        id: 'portal-id',
+        name: 'Test Portal',
+        shortcode: 'TESTPORTAL',
+        portalEnvironments: [],
+        portalStudies: [{
+          study: {
+            name: 'Test Study',
+            shortcode: 'STUDYSHORTCODE',
+            studyEnvironments: [{
+              id: 'test-study-env-id'
+            }]
+          }
+        }]
+      }
+    })
+  }
+})
+
+jest.mock('../providers/UserProvider', () => {
+  return {
+    useUser: () => ({
+      enrollees: [{
+        id: 'enrollee-id',
+        shortcode: 'ENSHORTCODE',
+        consented: false,
+        studyEnvironmentId: 'test-study-env-id',
+        participantTasks: [{
+          id: 'task-id',
+          blocksHub: true,
+          createdAt: 0,
+          enrolleeId: 'enrollee-id',
+          portalParticipantUserId: 'portal-participant-user-id',
+          status: 'NEW',
+          studyEnvironmentId: 'test-study-env-id',
+          taskType: 'CONSENT',
+          targetName: 'Test Survey',
+          targetStableId: 'test_survey_id',
+          targetAssignedVersion: 0,
+          taskOrder: 0
+        }]
+      }]
+    })
```

Comment:
using the mockEnrollee function will help make this test more robust against future changes tot he enrollee schema.

```suggestion
    useUser: () => ({
      enrollees: [{
        ...mockEnrollee(),
        consented: false,
        studyEnvironmentId: 'test-study-env-id',
        participantTasks: [{
          id: 'task-id',
          blocksHub: true,
          createdAt: 0,
          enrolleeId: 'enrollee-id',
          portalParticipantUserId: 'portal-participant-user-id',
          status: 'NEW',
          studyEnvironmentId: 'test-study-env-id',
          taskType: 'CONSENT',
          targetName: 'Test Survey',
          targetStableId: 'test_survey_id',
          targetAssignedVersion: 0,
          taskOrder: 0
        }]
      }]
    })
```

Reply:
It looks like this would require moving `mocking-utils` out of `ui-admin` and into `ui-core`. I think it's worth having shared test utils across the participant and admin UIs but from a quick glance it looks like it's too large of a refactor to have in this PR.